src/pages/subreddit/Subreddit.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import axios from 'axios';
+import './Subreddit-module.css';
+import Header from '../../componenten/header/Header';
+import { ReactComponent as BackIcon } from '../../assets/back.svg';
+
+function Subreddit() {
+    const [details, setDetails] = useState({});
```

Comment:
Ik de default value aanpassen naar `undefined` zodat je je check hier onderin makkelijker kan doen
```suggestion
    const [details, setDetails] = useState();
```